src/shogun/mathematics/linalg/GPUMemoryViennaCL.h

```diff
@@ -60,6 +60,17 @@ struct GPUMemoryViennaCL : public GPUMemoryBase<T>
 		init();
 	};
 
+	/** Creates a new vector
+	 *
```

Comment:
Such changes should be in separate patches, always try to split "operations" from "backend changes"